zhmccli/_cmd_imageprofile.py

```diff
@@ -0,0 +1,1868 @@
+        'initial-ziip-processing-weight': 'initial-ziip-processing-weight',
+        'minimum-ziip-processing-weight': 'minimum-ziip-processing-weight',
+        'maximum-ziip-processing-weight': 'maximum-ziip-processing-weight',
+        'ziip-processing-weight-capped':
+            'initial-ziip-processing-weight-capped',
+        'absolute-ziip-capping': None,
+        'dedicated-ziip-processors': 'number-dedicated-ziip-processors',
+        'reserved-dedicated-ziip-processors':
+            'number-reserved-dedicated-ziip-processors',
+        'shared-icf-processors': 'number-shared-icf-processors',
+        'reserved-shared-icf-processors':
+            'number-reserved-shared-icf-processors',
+        'initial-icf-processing-weight':
+            'initial-internal-cf-processing-weight',
+        'minimum-icf-processing-weight':
+            'minimum-internal-cf-processing-weight',
+        'maximum-icf-processing-weight':
+            'maximum-internal-cf-processing-weight',
+        'icf-processing-weight-capped':
+            'initial-internal-cf-processing-weight-capped',
+        'absolute-icf-capping': None,
+        'dedicated-icf-processors': 'number-dedicated-icf-processors',
+        'reserved-dedicated-icf-processors':
+            'number-reserved-dedicated-icf-processors',
+        'central-storage-origin': 'user-specified-central-storage-origin',
+        'access-basic-cpu-counter': 'basic-cpu-counter-authorization-control',
+        'access-problem-state-cpu-counter':
+            'problem-state-cpu-counter-authorization-control',
+        'access-crypto-activity-cpu-counter':
+            'crypto-activity-cpu-counter-authorization-control',
+        'access-extended-cpu-counter':
+            'extended-cpu-counter-authorization-control',
+        'access-coprocessor-cpu-counter':
+            'coprocessor-cpu-counter-authorization-control',
+        'access-diagnostic-sampling':
+            'diagnostic-sampling-authorization-control',
+        'access-basic-cpu-sampling': 'basic-cpu-sampling-authorization-control',
+        'permit-aes-key-import': 'permit-aes-key-import-functions',
+        'permit-des-key-import': 'permit-des-key-import-functions',
+        'permit-ecc-key-import': 'permit-ecc-key-import-functions',
+        'access-global-performance-data':
+            'global-performance-data-authorization-control',
+        'permit-global-iocds': 'io-configuration-authorization-control',
+        'permit-cross-partition-authority':
+            'cross-partition-authority-authorization-control',
+        'permit-bcpii-send-commands': 'security-bcpii-send-commands',
+        'permit-bcpii-receive-commands': 'security-bcpii-receive-commands',
+        'bcpii-receive-partitions': 'security-bcpii-receive-partition-list',
+        'time-offset-direction': 'time-offset-increase-decrease',
+    }
+
+    org_options = original_options(options)
+    if org_options['copy-name'] is not None:
```

Comment:
I wonder whether it is a good idea to copy the properties from the default profile ourselves. Also, since there is the profile named "DEFAULT" when --copy-name is not specified, it is inconsistent to copy only the profile when  --copy-name is specified, but not the DEFAULT profile when it is not specified.

Reply:
I removed the client-side copying.

Please review that approach.